src/pykiso/lib/robot_framework/uds_auxiliary.py

```diff
@@ -178,19 +175,13 @@ def start_tester_present_sender(self, period: int, aux_alias) -> None:
         :param aux_alias: auxiliary's alias
         """
         aux = self._get_aux(aux_alias)
-        self.sender = threading.Thread(
-            name="TesterPresentSender",
-            target=aux._sender_run,
-            args=(period, self.sender_stop_event),
-        )
-        self.sender.start()
+        aux.start_tester_present_sender(period)
```

Comment:
why is start_tester_present_sender not a lib function like  def stop_tester_present_sender ??